client/src/components/Donation/utils.ts

```diff
@@ -0,0 +1,7 @@
+const numToCommas = (num: number) => Intl.NumberFormat('en-US').format(num);
```

Comment:
Not blocking for this PR, but something to consider: can/should we let the browser default to formatting according to its location, rather than enforcing the US format?